content/zh/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -274,10 +274,6 @@ resource limits, see the
 ## Monitoring compute resource usage
 
 The resource usage of a Pod is reported as part of the Pod status.
-
-If [optional monitoring](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/cluster-monitoring/README.md)
```

Comment:
Only modify comments here will cause inconsistency for a translation doc.

Reply:
Good catch, I've removed the translated text too.